src/corehost/common/pal.windows.cpp

```diff
@@ -227,7 +240,44 @@ bool pal::get_default_installation_dir(pal::string_t* recv)
     }
 
     append_path(recv, _X("dotnet"));
+
+    return true;
+}
+
+bool pal::get_sdk_self_registered_dir(pal::string_t* recv)
+{
+#if _TARGET_ARM_
+    //  Self-registered SDK installation directory is not supported for win-arm
```

Comment:
Is there a fundamental reason why this cannot be supported on ARM, or are we scoping it out?

Reply:
@nguerrera 
The 'RegGetValueW' symbol does not resolve in the win-arm compiler - plus the self-registered location is a feature of the native SDK bundle installer.